packages/camera/camera_android/test/android_camera_test.dart

```diff
@@ -1119,3 +1121,10 @@ void main() {
     });
   });
 }
+
+// Temporary hack to work around changes in https://github.com/flutter/flutter/pull/116937
+// This should be removed once the relevant changes makes it to stable.
+// TODO(ianh): look into this in Q2 2023.
+T _ambiguate<T extends Object>(T? value) {
+  return value!;
```

Comment:
Is it intentional that this is the reverse of what we did last time? We did `T`->`T?` and then kept the `!` at the call site before, which seems more explicit about what's happening at the call site. If we want this behavior instead I think we should call this something that sounds dangerous, since it's force-unwrapping (e.g., `_forceUnwrapIfNullable`).

Reply:
@Hixie Ping on this question.

Reply:
no, definitely not intentional. i'll reverse it.